scripts/init-submodules-no-riscv-tools.sh

```diff
@@ -59,13 +40,6 @@ git config --unset submodule.generators/sha3.update
 # Shallow clone by default (recursive clone would get linux)
 git submodule update --init generators/sha3
 
-if [ $NO_FIRESIM = false ]; then
-  echo "initializing firesim"
-  # Renable firesim and init only the required submodules to provide
-  # all required scala deps, without doing a full build-setup
-  git config --unset submodule.sims/firesim.update
-  git submodule update --init sims/firesim
-  git -C sims/firesim submodule update --init sim/midas
-  git -C sims/firesim submodule update --init --recursive sw/firesim-software
```

Comment:
Until I move firemarshal to chipyard, we'll need to shallow clone firesim-software as well (it's symlinked into chipyard/sw/firemarshal). If this is targeting the Friday release, you'll need to add the shallow clone. If this is scheduled for after the release, you may as well wait for me to move Marshal.

Reply:
Does anything at the Chipyard top level mandate FireMarshal?  If someone wants to use FireMarshal, s/he would need to manually run its init-submodules.sh script, so it isn't an onerous expectation to have to initialize FireMarshal explicitly beforehand.

Reply:
(Or rely on the firesim-setup.sh script to do so.)